src/oc-id/app/controllers/profiles_controller.rb

```diff
@@ -41,6 +49,32 @@ def change_password
   end
 
   #
+  # GET /id/profile/change_email
+  #
+  # Changes the user's email.
+  #
+  # This should really be a PUT but we are using GET to make it easier to access
+  # the end-point from an email without having to dive into javascript.
```

Comment:
For more context, it might be worth mentioning that the initiating action is a form submit/POST, and that this GET is completion/verification of that action